### PR TITLE
Feature/project page

### PR DESCRIPTION
**Pull request by ashwin1321**
project section

### CODE REVIEW

File: src/assets/homeIcon/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import boy from "./boy.png";
```
Comment (rabin245) [inline src/assets/homeIcon/index.js:1]: unnecessary exporting. redundant task.

File: src/components/ProjectItem/Project.module.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  box-shadow: 0 2px 8px rgba(32, 28, 17, 0.26);
+  transition: all 0.3s ease-out;
+}
+
+.card:hover {
+  transform: scale(1.1);
+  box-shadow: 0 2px 8px rgba(22, 21, 21, 0.46);
+  margin: 5px;
+  background-color: #fdf0e4;
+}
```
Comment (rabin245) [inline src/components/ProjectItem/Project.module.css]: this needs refactoring. move all the cards to the same component.

File: src/pages/About/index.jsx [hunk 3]
```diff
@@ -1,10 +1,29 @@
...
+            <div className={Style.aboutSection}>
+                <div className={Style.img}>
+                    <img src={aboutboy} alt="boy image" className={Style.img1} />
+                </div>
+                <div className={Style.content}>
+                    <div className={Style.name} >Ashwin Khatiwada</div>
+                    <div className={Style.h1}>Software Engineer Intern/ Relaxed</div>
+                    <div className={Style.p}> Highly motivated and skilled individual with a passion for the tech industry seeking to become a good human. Enjoys hobbies such as playing guitar, hiking, exploring, and playing video games to stay inspired and focused. Proven ability to approach challenges with a clear mind and excited to join a team that values growth and development.</div>
+                    <div className={Style.buttons}>
+                        <a href="../../../public/ashwinCV.pdf">
```
Comment (rabin245) [inline src/pages/About/index.jsx:18]: dont need to go back to prev directory to access public folder and Move the assets to the assets folder.

File: src/components/ProjectItem/Project.module.css (hunk 2 above)
Comment (ashwin1321) [reply src/components/ProjectItem/Project.module.css]: done

File: src/components/Form/ContactForm.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import Style from "./ContactForm.module.css";
+import { GrMail } from "react-icons/gr";
+import Button from "../../components/Button";
+
+const ContactForm = () => {
+
```
Comment (rabin245) [inline src/components/Form/ContactForm.jsx]: remove app logic

Comment (rabin245) [reply src/components/Form/ContactForm.jsx]: move this component to the contact page

File: src/pages/Skills/index.jsx [hunk 5]
```diff
@@ -1,8 +1,46 @@
...
-        <div ref={skillsRef}>index</div>
+        <div ref={skillsRef} className={Style.skillSection}>
+            <div className={Style.section}>
+                <div className={Style.heading}>
+                    Skills and Experience
+                </div>
+
+                <div className={Style.content}>
+                    <div className={Style.skills}>
+                        <div className={Style.skill}>
```
Comment (rabin245) [inline src/pages/Skills/index.jsx]: move this div to its own component

**Review by rabin245 — CHANGES_REQUESTED** [review]
change the files as requested.

File: src/pages/Skills/index.jsx (hunk 5 above)
Comment (ashwin1321) [reply src/pages/Skills/index.jsx]: done

**Review by rabin245 — APPROVED** [review]
Looks good to merge.

**Pull request merged**